portal/server/controllers/utilisation-report-service/utilisation-report-upload/index.js

```diff
@@ -10,22 +10,30 @@ const { getReportAndUserDetails } = require('./utilisation-report-details');
  * the year and the report period with format 'MMMM yyyy'
  * @param {string} userToken - Token to validate session
  * @param {string} bankId - ID of the bank
- * @returns {Promise<{ month: number; year: number; reportPeriod: string }[]>}
+ * @returns {Promise<{ startMonth: number, startYear: number, reportPeriodString: string }[]>}
  */
 const getDueReportDates = async (userToken, bankId) => {
   const dueReports = await api.getDueReportDatesByBank(userToken, bankId);
   return dueReports.map((dueReport) => {
-    const { month, year } = dueReport;
-    const reportPeriod = format(new Date(year, month - 1), 'MMMM yyyy');
-    return { ...dueReport, reportPeriod };
+    const { startMonth, startYear } = dueReport;
+    const reportPeriodString = format(new Date(startYear, startMonth - 1), 'MMMM yyyy');
+    return { ...dueReport, reportPeriodString };
```

Comment:
For other formatted dates we've used something like `formattedReportPeriod` elsewhere. I think this also helps differentiate it from something that is just a stringified version (e.g. `JSON.stringify(reportPeriod)`) which I think we have done elsewhere and also named `reportPeriodString`.